src/main/resources/reference.conf

```diff
@@ -13,6 +13,22 @@ kamino {
   # Where to POST data to.
   ingestion-api = "https://ingestion.kamino.io/v1"
 
+  # Define proxy to be used for reaching Kamino API.
+  proxy {
+
+    # Specify proxy type to be used.
+    #   - system; JVM configured proxy
+    #   - http
+    #   - socks
+    type = "system"
```

Comment:
Regarding options, since Kamino only accepts data over HTTPS, should the proxy be a HTTPS proxy as well?